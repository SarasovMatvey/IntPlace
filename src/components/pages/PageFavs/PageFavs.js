import { Container } from "semantic-ui-react";
import FavPlaces from "../../organisms/FavPlaces";
import BaseTemplate from "../../templates/BaseTemplate";

function PageFavs() {
  return (
    <BaseTemplate>
      <Container>
        <FavPlaces />
      </Container>
    </BaseTemplate>
  );
}

export default PageFavs;
