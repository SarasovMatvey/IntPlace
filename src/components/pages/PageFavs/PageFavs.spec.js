import PageFavs from "./PageFavs";

jest.mock("../../../api/sygic.js", () => ({}));

const setUp = props => shallow(<PageFavs {...props} />);

describe("PageFavs", () => {
  let component;

  beforeEach(() => {
    component = setUp();
  });

  it("should renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
