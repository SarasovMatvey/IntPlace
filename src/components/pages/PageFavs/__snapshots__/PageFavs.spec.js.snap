// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFavs should renders properly 1`] = `
<BaseTemplate>
  <Container>
    <FavPlaces />
  </Container>
</BaseTemplate>
`;
